CONTRIBUTING.rst

```diff
@@ -998,10 +998,12 @@ prepare such packages on your own easily.
 
 * Enter the ``backport_packages`` directory
 
+* Run ``python setup_backport_packages.py prepare`` to update deprecated modules & paths.
+
 * Usually you only build some of the providers package. The ``providers`` directory is separated into
   separate providers. You can see the list of all available providers by running
   ``python setup_backport_packages.py list-backport-packages``. You can build the backport package
-  by running ``python setup.py <PROVIDER_NAME> bdist_wheel``. Note that there
+  by running ``python setup_backport_packages.py <PROVIDER_NAME> bdist_wheel``. Note that there
```

Comment:
I think we should encourage people to use one-step-only:
```
scripts/ci/ci_prepare_packages.sh packageA packageB packageC
```
WDYT?

Reply:
Yes. We should do it via script as it additionally cleans up whatever needs to be cleaned up. I've had a lot of troubles in the past trying to build package and forgot about cleaning.

Reply:
Fixed